druid/src/widget/checkbox.rs

```diff
@@ -69,16 +73,29 @@ impl Widget<bool> for Checkbox {
 
     fn layout(
         &mut self,
-        _layout_ctx: &mut LayoutCtx,
+        layout_ctx: &mut LayoutCtx,
         bc: &BoxConstraints,
-        _data: &bool,
+        data: &bool,
         env: &Env,
     ) -> Size {
         bc.debug_check("Checkbox");
 
+        // bc.constrain(Size::new(
+        //     env.get(theme::BASIC_WIDGET_HEIGHT),
+        //     env.get(theme::BASIC_WIDGET_HEIGHT),
+        // ))
+
+        let label_size = self.child_label.layout(layout_ctx, &bc, data, env);
+        let padding = 8.0;
```

Comment:
it feels like this should maybe be a constant somewhere, and at least shared between the radio button and the checkbox? That can be a separate PR, though.

`CONTROL_WIDGET_HORIZ_PADDING`?

Reply:
Yeah I'll do this along with fixing the padding of radio.